spec/integration/put_content/content_with_embedded_content_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe "PUT /v2/content when embedded content is provided" do
+  include_context "PutContent call"
+
+  context "with embedded content" do
+    let(:contact) { create(:edition, state: "published", content_store: "live", document_type: "contact") }
+    let(:document) { create(:document) }
+    let!(:link) { document.content_id }
+
+    before do
+      payload.merge!(document_type: "press_release", schema_name: "news_article", details: { body: "{{embed:contact:#{contact.document.content_id}}}" })
```

Comment:
We should probably have another test for when body is an array like `[{content_type: "text/html"}]`